irods/pool.py

```diff
@@ -44,28 +45,38 @@ def get_connection(self):
 
                 curr_time = datetime.datetime.now()
                 # If 'refresh_connection' flag is True and the connection was
-                # last used more than 'connection_refresh_time' seconds ago,
+                # created more than 'connection_refresh_time' seconds ago,
                 # release the connection (as its stale) and create a new one
-                if self.refresh_connection and (curr_time - conn.last_used_time).total_seconds() > self.connection_refresh_time:
-                    logger.debug('Connection has been idle more than {} seconds. Releasing the connection and creating a new one.'.format(self.connection_refresh_time))
+                logger.debug("In Pool's get_connection(): Connection with id {} was created {} seconds ago".format(id(conn), (curr_time - conn.create_time).total_seconds()))
```

Comment:
Debug statement to print the object id of connection and how long ago it was created.

Reply:
for dev only?   seems the same information is printed two lines below if it's too old

Reply:
Removed the line.